libs/libc/stream/lib_syslogstream.c

```diff
@@ -103,6 +102,37 @@ static void syslogstream_addchar(FAR struct lib_syslogstream_s *stream,
       syslogstream_flush(stream);
     }
 }
+
+int syslogstream_addstring(FAR struct lib_syslogstream_s *stream,
+                           FAR const char *buff, int len)
+{
+  FAR struct iob_s *iob = stream->iob;
+  uint16_t ret = 0;
```

Comment:
why use uint16_t not int

Reply:
iob->io_len is uint16,  but using int is better